Modelica/Resources/BuildProjects/CMake/CMakeLists.txt

```
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.15)
+
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+  if(WIN32 AND CMAKE_GENERATOR MATCHES "Visual Studio .*|NMake.*")
```

```suggestion
  if(WIN32)
```
I see no reason to limit the cmake generator here? For example, we sometimes use Ninja on Windows, we still want the static runtime.

---

Might be there to exclude mingw? But there should be a better way to achieve that.

---

Hm, that's taken from here: https://github.com/madler/zlib/pull/878/files where I tested it for zlib.